packages/kbn-test/src/es/test_es_cluster.ts

```diff
@@ -200,12 +201,14 @@ export function createTestEsCluster<
 
   const esArgs = assignArgs(defaultEsArgs, customEsArgs);
 
+  const testLicense: ArtifactLicense = getFips() === 1 ? 'trial' : license ? license : 'basic';
```

Comment:
Could you add a comment explaining the logic for `licence` pick in different cases?

Reply:
Added here https://github.com/elastic/kibana/pull/192632/commits/e4b6636674655b7be07022f2fb8fa7e6e66b5f46

Reply:
The more tests we can force to run = more assurance that we don't hit an OpenSSL error.


> For example, if a test is meant to verify that certain functionality is available in basic, running it in FIPS would be pointless.

Many tests default to `basic` and run fine when overridden with `trial`, if they fail due to configuration, I skip them for FIPS one way or another (depending on unit/integration/ftr). A test that is asserting something specific to a basic license in this case would fail and be skipped.

> Would it be possible to override this at the consumer/test-suite level and maybe even exclude suites that aren't relevant to trial, and hence FIPS?

There are so many tests, IMO, it would be very messy to have a FIPS check in every suite - and out of all the Unit/Integration there were so few that actually required a skip (usually due to OSS=false, not the license!)

To summarize, for the FIPS pipeline we just want to make sure we maximize our coverage to prevent FIPS related errors. Everyday CI is to assert working functionality.